packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/MultiFrameViewer.stories.js

```diff
@@ -41,6 +43,7 @@ const mockStore = {
   drawing: {
```

Comment:
This is cruft and not necessary for the story setup. I thought I had removed it with the SingleImageViewer, but must've missed that.